src/arrayMethodPush.js

```diff
@@ -7,8 +7,8 @@
  */
 
 function applyCustomPush() {
-  [].__proto__.push2 = function() {
-    // write code here
+  [].__proto__.push2 = function(...args) {
+    return this.push(...args);
```

Comment:
@mgrinko please write your own solution. Don't use standard method! 🥇

Reply:
I think it is overkill
I hope It is enough just to ask them